app/services/articles/feed.rb

```diff
@@ -82,6 +83,13 @@ def mix_default_and_more_random
       end
     end
 
+    # Test variation: the more comments a post has, the higher it's rated!
+    def more_comments
+      @comment_weight = 2
```

Comment:
General question: since this is an experiment, we may want to change values without redeploys. Do we maybe want to make them configurable somehow, maybe through a database table? The answer may well be no, as I'm not completely up to do with the field test work, but I just wanted to ask.

Reply:
I'd like @benhalpern to weigh in, but my opinion is that we should try to make the experiments immutable: Field Test doesn't record when we change the underlying experiment, so results from before and after a factor change in this equation would be all lumped together. I'd much rather split changes out with with multiple experiments like `comments_experiment_2x_value` and `comments_experiment_3x_value` versus having to remember the first thousand page views were at 2x value, the rest at 3x.

Reply:
Got it, thanks! 🙏 Now that you explain it, it makes total sense, this is purely a data gathering mechanism, so we need to keep the input params in mind. What I said makes more sense for experimental features, where we still may want to tweak how they behave.